data/src/main/java/camp/nextstep/edu/github/data/datasource/remote/GithubDataSource.kt

```diff
@@ -0,0 +1,23 @@
+package camp.nextstep.edu.github.data.datasource.remote
+
+import camp.nextstep.edu.github.data.datasource.RemoteDataSource
+import camp.nextstep.edu.github.data.datasource.remote.response.mappers.ResponseRepositoryMapper
+import camp.nextstep.edu.github.data.datasource.remote.service.GithubService
+import camp.nextstep.edu.github.data.di.MapperModule
+import camp.nextstep.edu.github.data.di.ServiceModule
+import camp.nextstep.edu.github.domain.model.Repository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+internal class GithubDataSource(
+    private val githubService: GithubService = ServiceModule.githubService,
+    private val responseRepositoryMapper: ResponseRepositoryMapper = MapperModule.responseRepositoryMapper
+) : RemoteDataSource {
+
+    override fun getReposotories(): Flow<List<Repository>> = flow {
+        val responseRepositoryList = githubService.getRepositories()
+        val repositoryList = responseRepositoryList.map(responseRepositoryMapper::mapToDomain)
+
+        emit(repositoryList)
```

Comment:
```suggestion
        githubService.getRepositories()
            .map(responseRepositoryMapper::mapToDomain)
            .also { emit(it) }
```
이부분은 메서드 체이닝으로 이렇게도 바꿔볼 수 있겠네요 🙂